src/main/java/com/balako/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,23 @@
 package com.balako.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @Size(min = 1, max = 255)
     private String title;
+    @NotNull
+    @Size(min = 1, max = 255)
     private String author;
+    @NotNull
     private String isbn;
```

Comment:
`isbn` has other validation rules which need to be performed. (You can use predefined annotation for that).